azure-pipelines.yml

```diff
@@ -250,31 +230,42 @@ jobs:
         - "/hw/top_englishbreakfast/Vchip_englishbreakfast_verilator"
 
 - job: execute_verilated_tests
-  displayName: Execute tests on the Verilated system (excl. slow tests)
+  displayName: Build and run tests on sim_verilator
   pool: ci-public
-  timeoutInMinutes: 120
-  dependsOn:
-    - chip_earlgrey_verilator
-    - sw_build
+  timeoutInMinutes: 180
+  dependsOn: lint
   steps:
   - template: ci/install-package-dependencies.yml
-  - template: ci/download-artifacts-template.yml
-    parameters:
-      downloadPartialBuildBinFrom:
-        - chip_earlgrey_verilator
-        - sw_build
+  - task: DownloadSecureFile@1
+    condition: eq(variables['Build.SourceBranchName'], 'master')
+    name: bazelCacheGcpKey
+    inputs:
+      secureFile: "bazel_cache_gcp_key.json"
+  - bash: echo "##vso[task.setvariable variable=bazelCacheGcpKeyPath]$(bazelCacheGcpKey.secureFilePath)"
+    condition: eq(variables['Build.SourceBranchName'], 'master')
+    displayName: Set the remote cache GCP key path
+  - bash: |
+      export GCP_BAZEL_CACHE_KEY=$(bazelCacheGcpKeyPath)
+      ci/bazelisk.sh test \
+        --build_tests_only=true \
+        --test_timeout=600,600,-1,-1 \
+        --local_test_jobs=4 \
+        --test_timeout_filters=short,moderate \
+        --test_tag_filters=verilator,-failing_verilator \
+        //sw/device/...
+        # TODO: include long tests after testing in CI
+        # --test_timeout_filters=short,moderate,long \
```

Comment:
Please could you put this in a script? I'd rather not have to copy-paste arcane runes from the YAML file whenever I want to reproduce a failure :-)

(Yes, I know that the Pytest command below was a bit arcane: I never got around to factoring it out!)

Reply:
Done!